ChallengesWithTestsMarkVII/ChallengesSet05.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ChallengesWithTestsMarkVII
+{
+    public class ChallengesSet05
+    {
+        public int GetNextNumberDivisibleByN(int startNumber, int n)
+        {
+            for (int i = startNumber; i < 10000000; i++)
+            {
+                if (i % n == 0 && i > startNumber)
```

Comment:
Consider changing the loop to `int i = startNumber + 1` and you wouldn't need to check `i > startNumber` each time